include/_antic.h

```diff
@@ -86,13 +133,43 @@ struct __antic {
 #define DL_MAP160x1x4   ((unsigned char) 14)     /* 4 colours, 160 pixel & 1 scanline per mode line (GR.15) */
 #define DL_MAP320x1x1   ((unsigned char) 15)     /* monochrome, 320 pixel & 1 scanline per mode line (GR.8) */
 
-/* modifiers on mode lines */
-#define DL_HSCROL(x)    ((unsigned char)((x) | 16))
-#define DL_VSCROL(x)    ((unsigned char)((x) | 32))
-#define DL_LMS(x)       ((unsigned char)((x) | 64))
+/* Equivalents, for people familiar with Atari 8-bit OS */
+#define DL_GRAPHICS0    DL_CHR40x8x1
+#define DL_GRAPHICS1    DL_CHR20x8x2
+#define DL_GRAPHICS2    DL_CHR20x16x2
+#define DL_GRAPHICS3    DL_MAP40x8x4
+#define DL_GRAPHICS4    DL_MAP80x4x2
+#define DL_GRAPHICS5    DL_MAP80x4x4
+#define DL_GRAPHICS6    DL_MAP160x2x2
+#define DL_GRAPHICS7    DL_MAP160x2x4
+#define DL_GRAPHICS8    DL_MAP320x1x1
+#define DL_GRAPHICS9    DL_MAP320x1x1  /* N.B.: GRAPHICS 9, 10, and 11 also involve GTIA's PRIOR register */
+#define DL_GRAPHICS10   DL_MAP320x1x1
+#define DL_GRAPHICS11   DL_MAP320x1x1
+#define DL_GRAPHICS12   DL_CHR40x8x4   /* N.B.: Atari 400/800 OS didn't have GRAPHICS 12 or 13 */
+#define DL_GRAPHICS13   DL_CHR40x16x4
+#define DL_GRAPHICS14   DL_MAP160x1x2
+#define DL_GRAPHICS15   DL_MAP160x1x4
+
+/* Atari 400/800 OS didn't have GRAPHICS 14 or 15, so they were known by "6+" and "7+" */
+#define DL_GRAPHICS6PLUS DL_GRAPHICS14
+#define DL_GRAPHICS7PLUS DL_GRAPHICS15
+
+/* Neither Atari 400/800 nor XL OS supported 10-scanline (descenders) text mode via CIO */
+#define DL_GRAPHICS0_DESCENDERS  DL_CHR40x10x1
+
+/* Modifiers to mode lines */
+#define DL_HSCROL(x)    ((unsigned char)((x) | 16)) /* enable smooth horizontal scrolling on this line; see HSCROL */
+#define DL_VSCROL(x)    ((unsigned char)((x) | 32)) /* enable smooth vertical scrolling on this line; see VSCROL */
+#define DL_LMS(x)       ((unsigned char)((x) | 64)) /* Load Memory Scan (next two bytes must be the LSB/MSB of the data to load */
+
+/* General modifier */
+#define DL_DLI(x)       ((unsigned char)((x) | 128)) /* enable Display List Interrupt on this mode line; requires NMIEN set to enable DLIs */
+
 
-/* general modifier */
-#define DL_DLI(x)       ((unsigned char)((x) | 128))
+/* Macros for the beginning and end of functions used as Display List Interrupts */
+#define DLI_START asm("pha"); asm("txa"); asm("pha"); asm("tya"); asm("pha");
+#define DLI_END asm("pla"); asm("tay"); asm("pla"); asm("tax"); asm("pla"); asm("rti");
```

Comment:
I've never used DLI interrupts from C. Could you explain their usage? (Maybe update the documentation.)

Reply:
I'm having a hard time to believe that there's a place in cc65 for something beside https://cc65.github.io/doc/funcref.html#ss3.218 for that purpose.

Reply:
I'll consider moving this to the function reference at some point, or perhaps just create some separate docs specific to cc65 for the Atari 8-bits (and hence also 5200).  I'm not an expert wrt this stuff, but I can probably gather some info from Action! language examples of DLIs, and the caveats.

Reply:
@billkendrick: Have you read my comment above (https://github.com/cc65/cc65/pull/831#discussion_r247509274) ?